inference-engine/src/vpu/common/src/ngraph/transformations/extract_dynamic_batch/slice_mat_mul.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "vpu/utils/error.hpp"
+#include "ngraph/opsets/opset5.hpp"
+#include "vpu/ngraph/transformations/extract_dynamic_batch/slice_mat_mul.hpp"
+
+namespace vpu {
+
+SliceConfiguration sliceMatMul(const ngraph::Node& node) {
+    VPU_THROW_UNLESS(node.get_input_size() == 2,  "Expecting operation {} to have {} inputs, got {}", node, 2, node.get_input_size());
```

Comment:
Target networks have only 2 inputs?
3 outputs is going to be supported?

Reply:
Didn't get your question. There I check Mat Mul operation has exactly 2 inputs. It must be so corresponding to specification of operation. There is no any additional restriction on count of network outputs since this function takes single mat mul into consideration.

Reply:
Yes, in the documentation there are only 2 inputs. Probably, option for 3 inputs is obsolete (there were examples of MatMul with 3 inputs some time ago).